odoo/__init__.py

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
+# ruff: noqa: E402, F401
```

Comment:
?

Reply:
Making ruff ignore unused imports and imports done in the middle of the file.